mkldnn.mk

```diff
@@ -22,10 +22,8 @@ ifeq ($(USE_MKLDNN), 1)
 	MXNET_INCLDIR = $(ROOTDIR)/include
 ifeq ($(UNAME_S), Darwin)
 	MKLDNN_LIBFILE = $(MKLDNNROOT)/lib/libmkldnn.1.dylib
-	MKLDNN_LIB64FILE = $(MKLDNNROOT)/lib64/libmkldnn.1.dylib
```

Comment:
why are the lib64 files being removed?

Reply:
To make the installation more deterministic. Now the binaries will be installed to MKLDNNROOT/lib while previously they will be installed to lib or lib64 or lib/x86_64-linux-gnu (as you can see in the issue report) according to different platforms.